SnapSheet/Classes/SnapSheetViewController.swift

```diff
@@ -22,7 +22,19 @@ public class SnapSheetViewController: UIViewController {
 	}
 
 	/// The state of the sheet
-	public var state: State = .closed
+	public var state: State = .closed {
+		didSet {
+			switch state {
+			case .open:
+				snap.snapPoint.y = sheetLayoutGuide.layoutFrame.minY
+			case .closed:
+				snap.snapPoint.y = sheetLayoutGuide.layoutFrame.maxY
+			case .dismissing, .moving:
```

Comment:
REQ: don't think we should allow setting to `.dismissing or `.moving`. I prefer making this private(set) and making `open()` and `close()` methods